CI - Fix replication log monitoring parser/timing failures

## Problem

`replication/repl_log_monitoring_test.py` failures recur in both BDB and LMDB CI. The representative failure is `test_replication_log_monitoring_advanced` raising from `lib389/repltools.py` because no valid replication log data was collected.

Recent signal from CI sweep:
- BDB `replication/repl_log_monitoring_test.py` failed in 5/20 sampled failed-main runs.
- LMDB `replication/repl_log_monitoring_test.py` failed in 3/20 sampled failed-main runs.
- Representative BDB run/job: https://github.com/389ds/389-ds-base/actions/runs/26732183826, job `78778958576`.
- Representative LMDB run/job: https://github.com/389ds/389-ds-base/actions/runs/26734200121, job `78784950931`.

Representative failure:

```text
ValueError: No valid replication data collected.
/usr/lib/python3.13/site-packages/lib389/repltools.py:1327
```

Captured logs show basic replication is working, so this may be parser behavior, timing, log-level setup, or missing/late log data rather than total replication failure.

## Proposed work

Investigate whether `ReplLogTool` is parsing the current log format correctly and whether the test waits for the advanced replication log entries before building results. Add diagnostics around collected log lines and timestamps if needed.

## Acceptance criteria

- `test_replication_log_monitoring_advanced` passes reliably in BDB and LMDB CI.
- Failures include actionable diagnostics about missing/invalid replication log data.
- Parser changes, if needed, are covered by a focused test for the observed log format.

Latest upstream CI state for: replication/repl_log_monitoring_test.py

LMDB
FAILED dirsrvtests/tests/suites/replication/repl_log_monitoring_test.py::test_replication_log_monitoring_multi_suffix

BDB
All tests pass

This commit addresses the failing replication/repl_log_monitoring_test.py on LMDB

--- a/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
+++ b/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
@@ -748,8 +748,13 @@ get_entry_type(WorkerQueueData_t *wqelmt, Slapi_DN *sdn)
     int len = SLAPI_ATTR_UNIQUEID_LENGTH;
     const char *ndn = slapi_sdn_get_ndn(sdn);
 
-    if (slapi_be_issuffix(be, sdn) && (wqelmt->wait_id == 1)) {
-        return DNRC_SUFFIX;
+    if (slapi_be_issuffix(be, sdn)) {
+        /* Is this is the root suffix entry */
+        if (wqelmt->wait_id == 1) {
+            return DNRC_SUFFIX;
+        }
+        /* Duplicate root suffix entry. */
+        return DNRC_BAD_SUFFIX_ID;
     }
     if (PL_strncasecmp(ndn, SLAPI_ATTR_UNIQUEID, len) || ndn[len] != '=') {
             return DNRC_OK;
@@ -4188,9 +4193,18 @@ dbmdb_bulk_producer(void *param)
                 thread_abort(info);
                 continue;
             case DNRC_BAD_SUFFIX_ID:
-                import_log_notice(job, SLAPI_LOG_ERR, "dbmdb_bulk_producer",
-                                  "Supplier's entry is inconsistent. (Suffix ID is %d instead of 1).", entry->id);
-                thread_abort(info);
+                import_log_notice(job, SLAPI_LOG_REPL, "dbmdb_bulk_producer",
+                                  "Skipping duplicate suffix entry \"%s\" (wire import id %d).",
+                                  slapi_entry_get_dn(entry->ep->ep_entry), entry->id);
+                free_bulk_queue_item(&entry);
+                entry = NULL;
+                continue;
+            case DNRC_NOPARENT_DN:
+                import_log_notice(job, SLAPI_LOG_REPL, "dbmdb_bulk_producer",
+                                  "Skipping entry \"%s\" with no extractable parent DN (wire import id %d).",
+                                  slapi_entry_get_dn(entry->ep->ep_entry), entry->id);
+                free_bulk_queue_item(&entry);
+                entry = NULL;
                 continue;
             case DNRC_NOPARENT_ID:
                 import_log_notice(job, SLAPI_LOG_ERR, "dbmdb_bulk_producer",

--- a/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
+++ b/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
@@ -749,7 +749,7 @@ get_entry_type(WorkerQueueData_t *wqelmt, Slapi_DN *sdn)
     const char *ndn = slapi_sdn_get_ndn(sdn);
 
     if (slapi_be_issuffix(be, sdn)) {
-        /* Is this is the root suffix entry */
+        /* Is this the root suffix entry */
         if (wqelmt->wait_id == 1) {
             return DNRC_SUFFIX;
         }
@@ -4193,14 +4193,14 @@ dbmdb_bulk_producer(void *param)
                 thread_abort(info);
                 continue;
             case DNRC_BAD_SUFFIX_ID:
-                import_log_notice(job, SLAPI_LOG_REPL, "dbmdb_bulk_producer",
+                import_log_notice(job, SLAPI_LOG_ERR, "dbmdb_bulk_producer",
                                   "Skipping duplicate suffix entry \"%s\" (wire import id %d).",
                                   slapi_entry_get_dn(entry->ep->ep_entry), entry->id);
                 free_bulk_queue_item(&entry);
                 entry = NULL;
                 continue;
             case DNRC_NOPARENT_DN:
-                import_log_notice(job, SLAPI_LOG_REPL, "dbmdb_bulk_producer",
+                import_log_notice(job, SLAPI_LOG_ERR, "dbmdb_bulk_producer",
                                   "Skipping entry \"%s\" with no extractable parent DN (wire import id %d).",
                                   slapi_entry_get_dn(entry->ep->ep_entry), entry->id);
                 free_bulk_queue_item(&entry);

--- a/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
+++ b/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
@@ -4193,14 +4193,14 @@ dbmdb_bulk_producer(void *param)
                 thread_abort(info);
                 continue;
             case DNRC_BAD_SUFFIX_ID:
-                import_log_notice(job, SLAPI_LOG_ERR, "dbmdb_bulk_producer",
+                import_log_notice(job, SLAPI_LOG_WARNING, "dbmdb_bulk_producer",
                                   "Skipping duplicate suffix entry \"%s\" (wire import id %d).",
                                   slapi_entry_get_dn(entry->ep->ep_entry), entry->id);
                 free_bulk_queue_item(&entry);
                 entry = NULL;
                 continue;
             case DNRC_NOPARENT_DN:
-                import_log_notice(job, SLAPI_LOG_ERR, "dbmdb_bulk_producer",
+                import_log_notice(job, SLAPI_LOG_WARNING, "dbmdb_bulk_producer",
                                   "Skipping entry \"%s\" with no extractable parent DN (wire import id %d).",
                                   slapi_entry_get_dn(entry->ep->ep_entry), entry->id);
                 free_bulk_queue_item(&entry);

